eq-author/src/App/questionPage/Design/PropertiesPanel/index.js

```diff
@@ -76,32 +75,30 @@ class PropertiesPanel extends React.Component {
         <PropertiesPaneBody>
           <ScrollPane>
             {!isEmpty(page) && (
-              <div>
-                <QuestionProperties page={page} />
-              </div>
+              <Accordion title="Optional fields">
+                <Padding>
+                  <QuestionProperties page={page} />
```

Comment:
Looks like you can still tab through the contents of this accordion panel even while it's hidden.